platform/launch/apis.go

```diff
@@ -0,0 +1,13 @@
+package launch
```

Comment:
Avoid having the launcher depend on the platform package, to keep the binary smaller